enh(hp::procurve::snmp): added uptime and spanning-tree modes + spanning-tree discovery rule (#3964)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/network-hp-standard-snmp.md

</TabItem>
<TabItem value="Not attached to a host template" label="Not attached to a host template">

| Service Alias              | Service Template                                       | Service Description                                                    | Discovery |
|:---------------------------|:-------------------------------------------------------|:-----------------------------------------------------------------------|:---------:|
| Cast-Generic-Name          | Net-HP-Standard-Cast-Generic-Name-SNMP-custom          | Check packets casting type of a network interface                      |           |
| Cast-Global                | Net-HP-Standard-Cast-Global-SNMP-custom                | Check packets casting type of multiple network interfaces              |     X     |

> | Packet-Errors-Generic-Name | Net-HP-Standard-Packet-Errors-Generic-Name-SNMP-custom | Check packets on errors/discards of a network interface                |           |

Reviewer: ```suggestion
| Packet-Errors-Generic-Name | Net-HP-Standard-Packet-Errors-Generic-Name-SNMP-custom | Check packets in error/discarded from a network interface                |           |
```

> | Packet-Errors-Global       | Net-HP-Standard-Packet-Errors-Global-SNMP-custom       | Check packets on errors/discards of multiple network interfaces        |     X     |

Reviewer: ```suggestion
| Packet-Errors-Global       | Net-HP-Standard-Packet-Errors-Global-SNMP-custom       | Check packets in error/discarded from multiple network interfaces        |     X     |
```
